io-engine/src/bdev/nexus/nexus_bdev.rs

```
@@ -1336,6 +1391,60 @@ pub async fn nexus_create_v2(
     }
 }
 
+async fn get_nexus_zone_info_from_children(children_devices: &mut Vec<(String, String)>) -> Result<ZoneInfo, Error> {
+    let mut zone_info = ZoneInfo::default();
+    let mut conventional = false;
+    let mut info_set = false;
+
+    for (_uri, device_name) in &*children_devices {
+        let dev = device_lookup(&device_name).unwrap();
```

should return an error